theme/lumo/vaadin-avatar-styles.html

```diff
@@ -14,12 +14,18 @@
         color: var(--lumo-secondary-text-color);
         background-color: var(--lumo-contrast-10pct);
         border-radius: 50%;
+        outline: none;
         cursor: default;
         user-select: none;
         -webkit-font-smoothing: antialiased;
         -moz-osx-font-smoothing: grayscale;
       }
 
+      /* TODO: does not work with <img> */
+      :host([focus-ring]) {
+        box-shadow: inset 0 0 0 2px var(--lumo-primary-color-50pct);
```

Comment:
Nit: we can create an issue right away. Just recorded 😅 
![focus-ring](https://user-images.githubusercontent.com/29372419/86237491-85150780-bba4-11ea-9425-2ab69564e3f4.gif)

Reply:
Thanks, created https://github.com/vaadin/vaadin-avatar/issues/25